src/components/info-tile/info-tile.tsx

```diff
@@ -0,0 +1,256 @@
+     */
+    @Prop({ reflect: true })
+    public prefix?: string;
+
+    /**
+     * A piece of text or number that is the main piece of information
+     * which the component is intended to visualize.
+     */
+    @Prop({ reflect: true })
+    public value: number | string;
+
+    /**
+     * A string of text that is visually placed after the value.
+     */
+    @Prop({ reflect: true })
+    public suffix?: string;
+
+    /**
+     * The `title` tag of the hyperlink, which can be used to
+     * provide additional information about the link.
+     * It improves accessibility both for sighted users
+     * and users with assistive technologies.
+     */
+    @Prop({ reflect: true })
+    public linkTitle?: string = null;
+
+    /**
+     * Set to `true` if info-tile is disabled.
+     */
+    @Prop({ reflect: true })
+    public disabled: boolean = false;
+
+    /**
+     * The url that the info-tile leads to.
+     */
+    @Prop({ reflect: true })
+    public href?: string = null;
+
+    /**
+     * Set to `true` if info-tile is clickable.
+     * This adds an elevated effect using a shadow, as well as `cursor: pointer`,
+     * which appears on `hover`. This is set to `true` automatically, when you
+     * provide an `href`.
+     * If the the component is used for visualization purposes only, this interactive
+     * visualization is not needed.
+     * But if you use this component somewhere without providing an `href` but you want
+     * it to be interactive for instance by adding `onClick`, make sure to set this
+     * to `true`.
+     */
+    @Prop({ reflect: true })
+    public clickable: boolean = false;
+
+    /**
+     * Where to load the linked URL, as the name for a browsing context:
+     * - `_self`: in the current browsing context. (Default)
+     * - `_blank`: in a new tab.
+     * - `_parent`: in the parent browsing context of the current one.
+     * If no parent, behaves as `_self`.
+     * - `_top`: the topmost browsing context (the "highest" context
+     * that's an ancestor of the current one). If no ancestors, behaves as `_self`.
+     */
+    @Prop({ reflect: true })
+    public target: '_self' | '_blank' | '_parent' | '_top' = '_self';
+
+    /**
+     * If specified, will display a notification badge
+     * on the info tile.
+     */
+    @Prop({ reflect: true })
+    public badge?: number | string;
+
+    /**
+     * -`value`: The value of the progress bar.
+     * - `maxValue`: The maximum value within the scale that the progress bar should visualize.
+     * Defaults to `100`.
+     * - `prefix`: The prefix which is displayed before the `progressValue`,
+     * must be a few characters characters long.
+     * `suffix`: The suffix which is displayed after the `value`, must be one or two characters long.
+     * Defaults to `%`
+     * - `percentageColors`: When set to `true`, makes the progress bar's percentage colorful.
+     * Colors change with intervals of 10%.
```

Comment:
These should be moved to the different properties on the `InfoTileProgress` interface

Reply:
I think they should be repeated there as well, you mean? 
Because if I only have them there, then they won't show up in the DOCs.
<img width="1039" alt="image" src="https://user-images.githubusercontent.com/35954987/203353161-8aca9041-a0b5-49f2-a169-040d31410616.png">

Reply:
is this a missing feature in Kompendium? Or is the correct way repeating the same info twice?

Reply:
I added a fixup, but the docs for `progress` do not compile properly anyway. I don't know what kind of dark magic is in the piece of markdown text which I wrote for them. 😭 🤮

Reply:
This is how more or less how all coding documentation tools work :smile: 
In here, you should add general documentation about the `progress` property and what it is used for. Each individual property on the `InfoTileProgress` interface needs to be documented on the interface itself. Otherwise, when a coder is using the interface they will not see any docs for each individual property and it will be weird to force them to go somewhere else to read about it.

If you want to duplicate the information here you can do that, but I would not recommend since it will lead to either more work if it needs to be updated, or it will be out of sync if someone forgets. There should be a link to the full documentation for the interface in the generated docs, but in your image it looks like there is not link generated :thinking:

Reply:
Yeah, a link should suffice. It would be very nice if such a link was autogenerated, but if it's not, we can add it manually.

+1 on not duplicating.

Reply:
Yeah, it had to be added there, and then there seems to be some problem with the descriptions, that prevent the output from being generated. I'm guessing it's got something to do with the `%` characters, since that was problematic elsewhere… I'm working on it 🙂

Reply:
Fixed. I'll resolve this thread now.